c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -1692,10 +1703,8 @@ bool body_reader_then_handler(JSContext *cx, HandleObject body_owner, HandleValu
   return JS::AddPromiseReactions(cx, promise, then_handler, catch_handler);
 }
 
-bool body_reader_catch_handler(JSContext *cx, HandleObject body_owner, HandleValue reader_val,
+bool body_reader_catch_handler(JSContext *cx, HandleObject body_owner, HandleValue extra,
                                CallArgs args) {
-  RootedObject reader(cx, &reader_val.toObject());
-
```

Comment:
`reader` wasn't used in the body of this function, and as `reader_val` is already a `HandleValue` that implies that it's rooted elsewhere.